cmd/kube-apiserver/app/options/options.go

```diff
@@ -85,4 +86,5 @@ func (s *APIServer) AddFlags(fs *pflag.FlagSet) {
 	// TODO: delete this flag as soon as we identify and fix all clients that send malformed updates, like #14126.
 	fs.BoolVar(&validation.RepairMalformedUpdates, "repair-malformed-updates", validation.RepairMalformedUpdates, "If true, server will do its best to fix the update request to pass the validation, e.g., setting empty UID in update request to its existing value. This flag can be turned off after we fix all the clients that send malformed updates.")
 	fs.BoolVar(&registry.EnableGarbageCollector, "enable-garbage-collector", false, "Enables the generic garbage collector. MUST be synced with the corresponding flag of the kube-controller-manager.")
+	fs.BoolVar(&s.EnableLeases, "enable-leases", false, "Enables leases on the API server.")
```

Comment:
Please name this according to what the leases do and make the description more complete. A user reading this would have no idea when they'd want to turn this on.